src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -792,6 +792,25 @@ public int CompareTo(String strB)
             return false;
         }
 
+        [Pure]
+        public Boolean EndsWith(Char value)
+        {
+            return EndsWith(new string(value, 1), StringComparison.CurrentCulture);
```

Comment:
I couldn't find a way to "culture-aware" compare char's , like CompareInfo do for String. Do we need to create a new API?